packages/react/src/components/Tabs/Tab.tsx

```diff
@@ -17,24 +17,24 @@ export type TabProps = {
  * @example
  * <Tabs.Tab value='1'>Tab 1</Tabs.Tab>
  */
-export const Tab = forwardRef<HTMLButtonElement, TabProps>((props, ref) => {
-  const { children, className, ...rest } = props;
-  const { size, ...useTabRest } = useTabItem(props);
+export const Tab = forwardRef<HTMLButtonElement, TabProps>(
+  function Tab(props, ref) {
```

Comment:
We usually have the full name here, so when you are not using compound component, it matches 1:1, just without the dot.
suggest going for `TabsTab`